doc/source/reshaping.rst

```diff
@@ -698,10 +703,124 @@ handling of NaN:
     In [3]: np.unique(x, return_inverse=True)[::-1]
     Out[3]: (array([3, 3, 0, 4, 1, 2]), array([nan, 3.14, inf, 'A', 'B'], dtype=object))
 
-
 .. note::
     If you just want to handle one column as a categorical variable (like R's factor),
     you can use  ``df["cat_col"] = pd.Categorical(df["col"])`` or
     ``df["cat_col"] = df["col"].astype("category")``. For full docs on :class:`~pandas.Categorical`,
     see the :ref:`Categorical introduction <categorical>` and the
     :ref:`API documentation <api.categorical>`.
+
+Frequently Asked Questions (and Examples)
+------------------
+
+In this section, we will review frequently asked questions and examples. The 
+column names and relevant column values are named to correspond with how this
+DataFrame will be pivoted in the answers below.
+
+.. ipython:: python
+
+   np.random.seed([3,1415])
```

Comment:
Any reason to choose a list as a seed here? Not saying there's anything wrong with it per se, just have always seen an int literal like `12345`

Reply:
No reason just idiosyncratic choice.